dojo/api_v2/views.py

```diff
@@ -1194,35 +1167,30 @@ class TestImportViewSet(prefetch.PrefetchListMixin,
-                Q(test__engagement__product__authorized_users__in=[self.request.user]) |
-                Q(test__engagement__product__prod_type__authorized_users__in=[self.request.user])
-            ).distinct()
-        else:
-            test_imports = Test_Import.objects.all()
-        return test_imports.prefetch_related(
+        return get_authorized_test_imports(Permissions.Test_View).prefetch_related(
                                         'test_import_finding_action_set',
-                                        'findings',
-                                        'findings__endpoints',
-                                        'findings__endpoint_status',
-                                        'findings__finding_meta',
-                                        'findings__jira_issue',
-                                        'findings__burprawrequestresponse_set',
-                                        'findings__jira_issue',
-                                        'findings__jira_issue',
-                                        'findings__jira_issue',
-                                        'findings__reviewers',
-                                        'findings__notes',
-                                        'findings__notes__author',
-                                        'findings__notes__history',
-                                        'findings__files',
-                                        'findings__images',
-                                        'findings__found_by',
-                                        'findings__tags',
-                                        'findings__risk_acceptance_set',
+                                        'findings_affected',
+                                        'findings_affected__endpoints',
+                                        'findings_affected__endpoint_status',
+                                        'findings_affected__finding_meta',
+                                        'findings_affected__jira_issue',
+                                        'findings_affected__burprawrequestresponse_set',
+                                        'findings_affected__jira_issue',
+                                        'findings_affected__jira_issue',
+                                        'findings_affected__jira_issue',
+                                        'findings_affected__reviewers',
+                                        'findings_affected__notes',
+                                        'findings_affected__notes__author',
+                                        'findings_affected__notes__history',
+                                        'findings_affected__files',
+                                        'findings_affected__images',
+                                        'findings_affected__found_by',
+                                        'findings_affected__tags',
+                                        'findings_affected__risk_acceptance_set',
```

Comment:
On the github diff it's not really clear what happened here, did you rename a field on the Test_Import model?

Reply:
The field on the Test_Import model is `findings_affected = models.ManyToManyField('Finding', through='Test_Import_Finding_Action')`, I haven't changed that. But when testing the API for Test_Import I got an error message that I fixed here.

Reply:
ah, so that field was actually renamed by me in the original PR but I forgot to update the prefetch in api. raised https://github.com/DefectDojo/django-DefectDojo/pull/4204 to fix that.